dir.props

```diff
@@ -13,6 +13,7 @@
     <PortableBuild Condition="'$(OS)' == 'Windows_NT'">true</PortableBuild>
     <PortableBuild Condition="'$(PortableBuild)' == ''">false</PortableBuild>
     <UseSystemLibraries Condition="'$(UseSystemLibraries)' == '' AND '$(PortableBuild)' != 'true'">true</UseSystemLibraries>
+    <UseSystemLibunwind Condition="'$(UseSystemLibunwind)' == '' AND '$(PortableBuild)' != 'true'">$(UseSystemLibraries)</UseSystemLibunwind>
```

Comment:
The `AND '$(PortableBuild)' != 'true'` doesn't look needed to me, and it reads better without it (because it matches the usual MSBuild pattern of defaulting a property to another):

```xml
    <UseSystemLibraries Condition="'$(UseSystemLibraries)' == '' AND '$(PortableBuild)' != 'true'">true</UseSystemLibraries>
    <UseSystemLibunwind Condition="'$(UseSystemLibunwind)' == ''">$(UseSystemLibraries)</UseSystemLibunwind>
```

Or is there a situation where `UseSystemLibraries` would be set to `true` by some other means during a portable build, and we *don't* want `UseSystemLibunwind` to automatically turn on?

Reply:
Good point. I can't imagine a *portable build* setting UseSystemLibraries to `true`. I will fix this.